SGX_Linux/README.md

```diff
@@ -10,6 +10,22 @@ application could be developed to completely avoid passing the WOLFSSL_CTX
 pointer between trusted and untrusted code for more security.
 
 ## Prerequisites
+Must have a copy of wolfSSL from either the website or github.
+
+To download a stable release head on over to https://www.wolfssl.com/download/
+and look for the wolfssl-x.x.x.zip (latest release)
+
+To grab a copy of the development master use `git clone https://github.com/wolfssl/wolfssl.git`
+NOTE: When cloning the github repository one must first run `./autogen.sh` from
+the root directory of wolfSSL to generate the configure script. Next run
+`./configure`.
```

Comment:
If adding these instruction we should have steps to create an empty options.h. Using ./config.status or similar to the document listed here https://github.com/wolfSSL/wolfssl/tree/master/IDE/LINUX-SGX noting to "touch options.h".

Reply:
@JacobBarthelmeh Done, thank you for the excellent feedback as always!